conf/evolutions/082-annotationsettings-volumeInterpolationAllowed.sql

```diff
@@ -0,0 +1,12 @@
+-- https://github.com/scalableminds/webknossos/pull/???
+
```

Comment:
```suggestion
```
I have come to believe that git blame should actually be enough to link evolutions back to PRs :) Let’s skip looking up and inserting the PR number here from now on. (No need to modify old evolution files, though)